metals-docs/src/main/scala/docs/WorksheetModifier.scala

```diff
@@ -0,0 +1,83 @@
+package docs
+
+import scala.meta.inputs.Input
+
+import mdoc.Reporter
+import mdoc.StringModifier
+
+class WorksheetModifier extends StringModifier {
+  override val name: String = "worksheet"
+
+  override def process(
+      info: String,
+      code: Input,
+      reporter: Reporter
+  ): String = {
+
+    val (howYouSeeEvaluations, howToHover) = info match {
+      case "vscode" =>
+        "as a decoration at the end of the line." ->
+          "hover on the decoration to expand the decoration."
+      case "vim" =>
+        """|differently depending on whether you are using Neovim or Vim. If 
+           |using Vim, you will see them appear as comments at the end of
+           |the line. If you're using Nvim you will see this as virtual text.
+           |Keep in mind that if you're using coc-metals with Nvim you'll need
+           |to make sure `codeLens.enable` is set to `true`.""".stripMargin ->
+          """|hover on the comment if you're in Vim or expand the virtual text by
+             |using the `coc-metals-expand-decoration` command. The default for this
+             |is:
+             |```vim
+             |nmap <Leader>ws <Plug>(coc-metals-expand-decoration)
+             |```
+             |""".stripMargin
+      case _ =>
+        "as a comment as the end of the line." -> "hover on the comment to expand."
+    }
+
+    s"""|## Worksheets
+        |
+        |Worksheets are a great way to explore an api, try out an idea, or code
+        |up an example and quickly see the evaluated expression or result. Behind
+        |the schens worksheets are powered by the great work done in
+        |[mdoc](https://scalameta.org/mdoc/).
+        |
+        |### Getting started with Worksheets
+        |
+        |To get started with a worksheet you can either use the `metals.new-scala-file`
+        |command and select *Worksheet* or create a file called `*.worksheet.sc`.
+        |This format is important as it is what tells Metals that it's meant to be
+        |treated as a worksheet and not just a Scala script. Where you create the
```

Comment:
`It also matters where you create the worksheet` ?

Reply:
I think both are actually correct.